webdriver-spec.html

```diff
@@ -5603,20 +5603,13 @@ <h2 data-dfn-for="NavigatorAutomationInformation">Interface</h2>
  <li><p>Wait until the user agent event loop has spun enough times to
   process the DOM events generated by the previous step.
 
- <li><p>If the click causes <a>navigation</a>:
+ <li><p>Perform implementation-defined steps to allow
+  any <a data-lt="navigation">navigations</a> triggered by the click
+  to start.
 
-  <ol>
-   <li><p>Run the <a>post-navigation checks</a>
-    and return its value if it is an <a>error</a>.
-
-   <li><p><a>Try</a> to <a>wait for navigation to complete</a>.
-  </ol>
-
-  <p class=example>Note that one may not know
-   if a click causes <a>navigation</a>.
-   It is safe to run the <a>post-navigation checks</a>
-   and <a>waiting for the navigation to complete</a>
-   even though navigation <em>did not</em> occur.
+ <li><p><a>Try</a> to <a>wait for navigation to complete</a>.
```

Comment:
Please note that there can be a delay in when the navigation actually get started. How should this be handled, and that we do not return too early? In Marionette we currently use a hardcoded delay of 200ms for now. Maybe we can spec something better?

Reply:
Also waiting in general is not such a great idea given that not all clicks will trigger a navigation.

Reply:
There ought to be an issue blocking L1 if this is to make it into the spec before it is frozen for REC next Thursday.

Reply:
Ok, I will file one.

Reply:
https://github.com/w3c/webdriver/issues/1046